
// try , catch ,finally ,throw , throws

public class ExceptionsDemo {

	public static void main(String[] args) {

		try {
	
			System.out.println("line 3 "+(10/0));  
		
		}
		catch (ArithmeticException e) {
		System.out.println("cannot div a number by zero .. which school ???");
		System.out.println(e.getMessage());
		System.out.println(e);
		System.out.println("------------");
		
		e.printStackTrace();
		
		}
		
		System.out.println("line 1");
		System.out.println("line 2");
	
	}

}
